scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/semanticdb/AttributesOps.scala

```diff
@@ -165,10 +165,11 @@ trait AttributesOps { self: DatabaseOps =>
 
                 names(mtree.pos) = symbol
                 if (mtree.isBinder) binders += mtree.pos
-                denotations(symbol) = gsym.toDenotation
+                denotations(symbol) = gsym.toDenotation(symbol)
```

Comment:
I'm not sure it's worth it to add complications to the signature of `toDenotation` just because we want to simplify debugging. Once I saw this change, I had to stop myself and see why you need to pass the symbol explicitly, and the answer was non-obvious. I wouldn't want to impose this comprehension cost on others reading this code (including us in the future).

How about we cache the results of `toSemantic` if you feel strongly about performing the computations only once?